documentation/scaling_recommendations.markdown

```diff
@@ -23,25 +22,14 @@ As with scaling any service, there are several possible performance and reliabil
 Bottleneck: Database Performance
 -----
 
-### Database Backend
-
-PuppetDB has two available database backends:
-
-* PostgreSQL
-* Embedded HSQLDB
-
-The PostgreSQL backend provides the best performance, and can be set
-up as described in the [configuration documentation][postgres].
-
-The embedded database is only suitable for up to about 100 Puppet
-nodes and [requires a significantly larger Java heap][ram].  It has
-also been deprecated, and will be removed in a future release.  Please
-see this [email][hsqldb_deprecation_mail] to the puppet-users list for
-further information.
-
 ### PostgreSQL Speed and Availability
 
-Using the PostgreSQL backend, PuppetDB will be limited by the performance of your Postgres server. You can increase performance by making sure your DB server has an extremely fast disk, plenty of RAM, a fast processor, and a fast network connection to your PuppetDB server. You may also need to look into database clustering and load balancing.
+PuppetDB will be limited by the performance of your Postgres
+server. You can increase performance by making sure your DB server has
+an extremely fast disk, plenty of RAM, a fast processor, and a fast
+network connection to your PuppetDB server. You may also need to look
+into database clustering and load balancing.  Note that the default
+PostgreSQL configuration on your system may be very conservative.
```

Comment:
@wkalt seems like we should get pg_tune into our scaling recommendations, is it something we can just put a paragraph on, or would it be a significant effort (i.e. separate page)?